content/_data/changelogs/weekly.yml

```diff
@@ -1785,6 +1785,34 @@
         - issue: 48754
         - url: https://groups.google.com/forum/#!topic/jenkinsci-dev/-VQ5YU0yYhg
           title: Retrospective thread in the developer mailing list
+- version: "2.101"
+  date: 2018-01-07
+  changes:
+    - type: rfe
+      pull: 3214
+      message: >
+        Log EOF exceptions on a <code>FINE</code> level instead of <code>WARNING</code> when the Jetty connection is closed by peer.
```

Comment:
Was this the annoying log spam e.g. when browsers stop loading the page? If so, 🎉

Reply:
yes, it was